pkg/agent/agent.go

```diff
@@ -530,27 +534,33 @@ func (f *Flows) onInterfaceEvent(iface ifaces.Interface, add bool) {
 		return
 	}
 	if add {
-		alog.WithField("interface", iface).Info("interface detected. trying to attach TCX hook")
-		if err := f.ebpf.AttachTCX(iface); err != nil {
-			alog.WithField("interface", iface).WithError(err).
-				Info("can't attach to TCx hook flow ebpfFetcher. fall back to use legacy TC hook")
-			if err := f.ebpf.Register(iface); err != nil {
-				alog.WithField("interface", iface).WithError(err).
-					Warn("can't register flow ebpfFetcher. Ignoring")
+		// TODO: use iface.HookType instead
```

Comment:
I noticed there's often this func invoked with mac=0:0:0:0:0:0 / ifindex=0 / netns=none, which ends up failing, I guess we should directly skip attaching in that case
(ifindex=0 isn't sufficient to skip if it comes with mac/netns)